server/textStrings/en.ts

```diff
@@ -14,6 +14,7 @@ export const strings: Record<string, Record<string, string>> = {
     missingSpoNoRecallRationale: 'You must explain your decision',
     missingLastName: 'Enter a last name',
     missingFirstName: 'Enter a first name',
+    missingFirstNames: 'Enter a first name(s)',
```

Comment:
Think this should be "Enter first name(s)" rather than "Enter a first name(s)", as per Figma.

Reply:
tks